sylvia-derive/src/input.rs

```diff
@@ -153,9 +154,15 @@ impl<'a> ImplInput<'a> {
             #exec
 
             #query
+
+            #entry_points
```

Comment:
I don't see where do you disable generation entry points? We cannot generate them when the feature flag for it is not enabled (or the other way).

Reply:
Here https://github.com/CosmWasm/sylvia/pull/90/files#diff-83fed236913cceb2202f2c224d7b2096cd8c557a9b9808153c34f27eb0ca33daR863

Reply:
Ok, I missed this line, its ok